internal/processorinfo/processor_count.go

```diff
@@ -0,0 +1,29 @@
+package processorinfo
+
+import (
+	"runtime"
+	"syscall"
+)
+
+var (
+	modkernel32 = syscall.NewLazyDLL("kernel32.dll")
```

Comment:
I would prefer we expose this using the normal `go generate` mksyscall_windows stuff.